irc/handlers.go

```diff
@@ -2469,7 +2469,13 @@ func userHandler(server *Server, client *Client, msg ircmsg.IrcMessage, rb *Resp
 
 	err := client.SetNames(msg.Params[0], msg.Params[3], false)
 	if err == errInvalidUsername {
-		rb.Add(nil, server.name, ERR_INVALIDUSERNAME, client.t("Malformed username"))
+		// if client's using a unicode nick or something weird, let's just set 'em up with a blank username instead.
```

Comment:
"blank" is a bit misleading

Reply:
oh yeah, I tried to just set a blank one but it didn't wanna work with me, heh. so ended up just using `user`. nice catch